scico/linop/radon_svmbir.py

```diff
@@ -230,15 +237,35 @@ def __init__(
         if prox_kwargs is None:
             prox_kwargs = dict()
 
+        default_prox_args = {"maxiter": 1000, "ctol": 0.001}
+        default_prox_args.update(prox_kwargs)
+
         svmbir_prox_args = dict()
-        if "maxiter" in prox_kwargs:
-            svmbir_prox_args["max_iterations"] = prox_kwargs["maxiter"]
-        if "ctol" in prox_kwargs:
-            svmbir_prox_args["stop_threshold"] = prox_kwargs["ctol"]
+        if "maxiter" in default_prox_args:
+            svmbir_prox_args["max_iterations"] = default_prox_args["maxiter"]
+        if "ctol" in default_prox_args:
+            svmbir_prox_args["stop_threshold"] = default_prox_args["ctol"]
```

Comment:
This approach makes sense for just two options to be passed to svmbir, but if that were ever to expand, I would suggest switching to a loop over a dict mapping our parameter names to svmbir ones.

Reply:
Thanks. I will keep that in mind as we change the svmbir interface.